ratis-proto/src/main/proto/Raft.proto

```diff
@@ -390,6 +390,7 @@ message SetConfigurationRequestProto {
 message TransferLeadershipRequestProto {
   RaftRpcRequestProto rpcRequest = 1;
   RaftPeerProto newLeader = 2;
+  uint64 timeoutInMs = 3;
```

Comment:
Similar to TransferLeadershipRequest, let's move it to RaftRpcRequestProto and name it "timeoutMs".

BTW, let's also move routingTable from RaftClientRequestProto to RaftRpcRequestProto.  The routingTable in RaftRpcRequestProto can be used for appendEntries later.  See if you want to do it here, or we may do it in a separated JIRA.

Reply:
@szetszwo Ok, I will do it in a separated JIRA.